en/docs/chapter_heap/summary.md

```diff
@@ -2,16 +2,16 @@
 
 ### Key review
 
-- A heap is a complete binary tree, which can be divided into a max heap and a min heap based on its property. The top element of a max (min) heap is the largest (smallest).
+- A heap is a complete binary tree, which can be divided into a max heap and a min heap based on its building property. The top element of a max (min) heap is the largest (smallest).
```

Comment:
optional (this may kind of break the accuracy of the translation but give a bit better clarity):
 The translation is accurate but could be refined for clarity. Consider
"A heap is a complete binary tree, which can be divided into a max heap and a min heap based on its building property. The top element of a max (min) heap is the largest (smallest)"
->
 "A heap is a complete binary tree that can be categorized as either a max heap or a min heap based on its building property."

Reply:
@yuelinxin Please address the comment thx!